assets/src/edit-story/components/canvas/frameElement.js

```diff
@@ -91,7 +92,9 @@ function FrameElement({ element }) {
       }}
     >
       <WithElementMask element={element} fill={true}>
-        {Frame && <Frame element={element} box={box} />}
+        <WithDropTarget element={element}>
```

Comment:
Not all types of element's can be a drop target. Will you translate it somewhere?